keps/sig-storage/3107-csi-nodeexpandsecret/kep.yaml

```diff
@@ -0,0 +1,43 @@
+title: SecretRef field addition to NodeExpandVolume request
+kep-number: 3107
+authors:
+  - "@humblec"
+owning-sig: sig-storage
+participating-sigs:
+  - sig-storage
+  - sig-api
+status: provisional
+creation-date: 2022-01-23
+reviewers:
+  - TBD
```

Comment:
You can add me under reviewers and approvers

Reply:
Thanks @msau42 , added :+1: